runtimes/python-3.10/requirements.txt

```diff
@@ -1 +1,2 @@
-Flask==2.0.1
\ No newline at end of file
+Flask==2.0.1
```

Comment:
We still kept Flask dependency

Reply:
Yep Flask is still providing the framework, we are just replacing the web server